ci-operator/step-registry/ipi/conf/tang/ipi-conf-tang-ref.yaml

```diff
@@ -0,0 +1,12 @@
+ref:
+  as: ipi-conf-tang
+  from: upi-installer
```

Comment:
Maybe we can replace the image with ` from: cli`

Reply:
In step `ipi-conf-tang`, require `jq` command, but it is not installed in cli image, so still needs `upi-installer` image here.

Reply:
Maybe we can use this one:
```
  from_image:
    namespace: ocp
    name: cli-jq
    tag: latest
```

Reply:
ah, let me update to use `cli-jq`

Reply:
get new error, no ssh command in image "cli-jq".
` /bin/bash: line 40: ssh: command not found `

Reply:
Per the error - `/bin/bash: line 40: ssh: command not found`, seem like cli-jq does not install ssh client, let us use:
```
  from_image:
    namespace: ocp
    name: "4.12"
    tag: upi-install
```
to replace `from: upi-installer` which has dependency on job definition.